### PR TITLE
README

### PR DESCRIPTION
**Pull request by subhojit777**

### CODE REVIEW

**Comment by subhojit777** [comment]
It would be great for new comers who want to contribute to the documentation.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Drupal Commerce Documentation
+
+## Local setup
+- Install [Python](https://www.python.org/downloads/)
+- If `pip` is not installed by default, install it from [here](https://pip.pypa.io/en/stable/installing/)
+- Execute `pip install sphinx sphinx-autobuild`
```
Comment (mglaman) [inline README.md]: You do not need this, only `pip install -r requirements.txt`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Drupal Commerce Documentation
+
+## Local setup
```
Comment (mglaman) [inline README.md]: Let's encourage people to use the GitHub UI to propose changes, and have them built in a PR by Platform.sh We should entter with this info and then say "but if you want to contribute heavily, we recommend..."

File: README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- If `pip` is not installed by default, install it from [here](https://pip.pypa.io/en/stable/installing/)
+- Execute `pip install sphinx sphinx-autobuild`
+- Fork this [documentation repository](https://github.com/drupalcommerce/commerce-docs)
+- Clone the forked repo
+- `cd <forked-repo>`
+- Execute `pip install -r requirements.txt`, you might need to execute this command with `sudo`
+- Execute `./build.sh`, this will generate documentation HTML inside `_build/html`
+- `cd` inside `_build/html`
+- Now you can view the documentation locally, you have these options:
+  - You have already installed Python, start `SimpleHTTPServer` using `python -m SimpleHTTPServer 8000`, and visit [http://0.0.0.0:8000/](http://0.0.0.0:8000/) from your browser
```
Comment (mglaman) [inline README.md]: We could just tell people to run `php -S localhost:8080 -t _build/html`, let them figure out anything else.

**Comment by subhojit777** [comment]
@mglaman Done with the changes. Please check now. The text is mostly copied from the Symfony documentation, hope you don't mind.

**Comment by mglaman** [comment]
@subhojit777 I like it! @bojanz thoughts?

**Pull request merged**